Kickstarter.xcodeproj/project.pbxproj

```diff
@@ -6985,6 +7016,7 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 12.4;
 				MARKETING_VERSION = 4.10.0;
 				MTL_ENABLE_DEBUG_INFO = NO;
+				ONLY_ACTIVE_ARCH = NO;
```

Comment:
We can revert these `ONLY_ACTIVE_ARCH` values being set explicitly to `NO`. Just select wherever they've been set in Xcode and hit delete I think.